frontend/src/components/Admin/AddUser/AddUser.tsx

```diff
@@ -125,6 +146,46 @@ export default function AddUser(): JSX.Element {
     });
   }
 
+  async function uploadUsersCsv(csvFile: File | undefined): Promise<void> {
+    if (csvFile) {
+      const csv = await csvFile.text();
+      const json = await csvtojson().fromString(csv);
+      const allNetIds = allMembers.map((m) => m.netid);
+      const members: IdolMember[] = [];
+      if (json[0].email && json[0].role) {
```

Comment:
The requirement for `role` should only be enforced for new members. For existing members, we should just keep the role the same if it's unspecified.